src/Diagnostics/Roslyn/Core/ApiDesign/DeclarePublicAPIAnalyzer.cs

```diff
@@ -110,7 +110,7 @@ public override void Initialize(AnalysisContext context)
                         return;
                     }
 
-                    var publicApiName = symbol.ToDisplayString(PublicApiFormat);
+                    string publicApiName = GetPublicApiName(symbol);
```

Comment:
I considered that, but then the call to `symbol.ToDisplayString(PublicApiFormat)` would put the return type at the beginning of the string. Since the code fix does a simple alphabetical sort of the lines in PublicAPI.txt, the symbols would then be sorted primarily by return type, rather than by type and member name.